lib/ansible/plugins/doc_fragments/docker.py

```diff
@@ -54,14 +54,14 @@ class ModuleDocFragment(object):
             - Path to the client's TLS certificate file.
             - If the value is not specified in the task and the environment variable C(DOCKER_CERT_PATH) is set,
               the file C(cert.pem) from the directory specified in the environment variable C(DOCKER_CERT_PATH) will be used.
-        type: str
+        type: path
```

Comment:
Why don't you also change `cacert_path` (one above)? Also, now the types differ from the ones here: https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/docker/common.py#L83-L85

Reply:
Fixed.